libs/SalesforceAnalytics/build.gradle

```diff
@@ -14,7 +14,7 @@ apply plugin: 'com.github.dcendents.android-maven'
 apply plugin: 'com.jfrog.bintray'
 
 dependencies {
-    compile 'com.squareup:tape:1.2.3'
+    api 'com.squareup:tape:1.2.3'
```

Comment:
`api` for third-party libraries that may or may not have sources and are pulled from `Maven`.